lib/game_runner.rb

```diff
@@ -27,6 +27,7 @@ def player_input
   def run
     introduce_game
     player_1_turn
+    computer_turn
```

Comment:
Now that you have this `computer_turn` method the `1` doesn't make much sense. What do you think about renaming it?

Reply:
✔️ I like it!